JitsiConference.js

```diff
@@ -2407,6 +2407,31 @@ JitsiConference.prototype.isCallstatsEnabled = function() {
     return this.statistics.isCallstatsEnabled();
 };
 
+/**
+ * Finds the SSRC of a given track
+ *
+ * @param track
+ * @returns {number|undefined} the SSRC of the specificed track, otherwise undefined.
+ */
+JitsiConference.prototype.getSsrcByTrack = function(track) {
+    const isP2P = track.isP2P;
```

Comment:
It seems this is not available for local tracks where you are actually using it. Am I missing something?

Maybe you can obtain the peer connection by using `getActivePeerConnection`?

Reply:
@hristoterezov thanks for taking a look!

> It seems this is not available for local tracks where you are actually using it.

Hmm.. You mean the `isP2P` field, right? That's interesting.. I don't understand all the interactions here, I've simply copied the ssrc lookup code from the `_onTrackAttach` function below.

Here's my concern with your suggestion: if the `track` parameter refers to a local P2P track and the active connection is the JVB one, then the function won't be able to find the track ssrc.

Unfortunately, the current code suffers from similar issues as you've correctly pointed out: if the `track` parameter refers to a local P2P track we'll never search for the SSRC in the P2P peer connection because `track.isP2P` is undefined (falsy). I wonder if this situation causes issues in the `_onTrackAttach` function below?

Reply:
Only remote tracks are either p2p or not p2p. Local tracks can belong to many peerconnections.

Reply:
This looks good in general, but I would also suggest using `getActivePeerConnection`

Reply:
Thanks guys! I patched my code the way you suggested.